services/k8s/apps/deployments.yml

```diff
@@ -0,0 +1,111 @@
+name: Deployments
+description: >-
+  Control Kubernetes Deployment objects.
+scope: CRITICAL
+notes: >-
+  Deployments are declarative updates to Kubernetes Pods and ReplicaSets.
+  They provide an abstraction layer that allows services to discover and connect to pods 
+  running within a Deployment. Applications can scale, load balance, and seamlessly handle changes 
+  in the underlying pod instances. Access to Deployments is sensitive because they are
+  a primary interface for controlling applications running on Kubernetes. A Deployment specifies the
+  container image to run, and may lead to arbitrary code execution in the cluster. A good mitigation
+  strategy is to make Kubernetes clusters private, which effectively disables pulling images over the internet.
+  Binary Authorization in GKE is an additional layer of security to defend against supply chain attacks.
```

Comment:
GKE reference here, though this may be reasonable to keep

Reply:
I actually removed it